src/Microsoft.DotNet.Darc/Darc/Operations/VirtualMonoRepo/CodeFlowOperation.cs

```diff
@@ -184,6 +209,85 @@ private async Task<Build> GetBuildAsync(NativePath sourceRepoPath)
         return build;
     }
 
+    protected async Task<bool> FlowForwardAsync(
+        ILocalGitRepo productRepo,
+        Build build,
+        Codeflow currentFlow,
+        SourceMapping mapping,
+        string headBranch,
+        IReadOnlyList<string> excludedAssets,
+        string? targetRepoUri,
+        CancellationToken cancellationToken)
+    {
+        try
+        {
+            CodeFlowResult result = await _forwardFlower.FlowForwardAsync(
+                mapping.Name,
+                productRepo.Path,
+                build,
+                excludedAssets: excludedAssets,
+                headBranch,
+                headBranch,
+                targetRepoUri ?? _vmrInfo.VmrPath,
```

Comment:
Can you take this from my branch?

If `targetRepoUri` is `null`, we'd just put there the first remote we find in the VMR dir:

```
var vmr = _localGitRepoFactory.Create(_vmrInfo.VmrPath);
(await vmr.GetRemotesAsync()).First().Uri
```